administrator/language/en-GB/en-GB.plg_behaviour_taggable.ini

```diff
@@ -0,0 +1,7 @@
+; Joomla! Project
+; Copyright (C) 2005 - 2015 Open Source Matters. All rights reserved.
+; License GNU General Public License version 2 or later; see LICENSE.txt, see LICENSE.php
+; Note : All ini files need to be saved as UTF-8
+
+PLG_BEHAVIOUR_TAGGABLE="Behaviour - Taggable"
+PLG_BEHAVIOUR_TAGGABLE_XML_DESCRIPTION="Allows content items to be tagged."
```

Comment:
In the Orthogonal Component Structure, this does not apply to content only. It should better read
`PLG_BEHAVIOUR_TAGGABLE_XML_DESCRIPTION="Allows items to be tagged."`